CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 
 ## 18.03.1-ce (2018-04-DD)
 
+### Client
+
+- Fix error with merge composefile with networks [docker/cli#983](https://github.com/docker/cli/pull/983)
```

Comment:
consider replacing `composefile` with `compose file` for consistency with the docs